test/test.js

```diff
@@ -152,3 +152,22 @@ describe('Parsing RGB Rasters', function() {
     });
   });
 });
+
```

Comment:
@DanielJDufour I attempted to add a test here, but it's currently failing with an obscure error:

```
(node:94877) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'protocol' of undefined
    at Object.http.request (webpack://GeoRaster/./node_modules/stream-http/index.js?:18:40)
    at Object.get (webpack://GeoRaster/./node_modules/stream-http/index.js?:43:17)
    at Object.https.get (webpack://GeoRaster/./node_modules/https-browserify/index.js?:17:19)
    at eval (webpack://GeoRaster/./node_modules/geotiff/dist/source.js?:728:96)
    at new Promise (<anonymous>)
    at _callee7$ (webpack://GeoRaster/./node_modules/geotiff/dist/source.js?:726:49)
    at tryCatch (webpack://GeoRaster/./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:62:40)
    at Generator.invoke [as _invoke] (webpack://GeoRaster/./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:296:22)
    at Generator.prototype.(anonymous function) [as next] (webpack://GeoRaster/./node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js?:114:21)
    at step (webpack://GeoRaster/./node_modules/babel-runtime/helpers/asyncToGenerator.js?:17:30)
    at eval (webpack://GeoRaster/./node_modules/babel-runtime/helpers/asyncToGenerator.js?:35:14)
```

Do you happen to have any ideas on what might be causing that? Seems like it fails as soon as geotiff.js tries to do an HTTP/HTTPS request.

Reply:
This is an issue with webpack trying to bundle libraries for making external calls.  I ran into a similar problem with geoblaze and solved it by basically moving the `node-fetch` library into webpack externals, so that it doesn't try to bundle it and it looks for it on the system when you run the relevant code in the bundle.  Here's my PR to fix that for geoblaze:
https://github.com/GeoTIFF/geoblaze/pull/151

It seems however that the problems comes from the `fromUrl` method of [geotiff.js](https://github.com/geotiffjs/geotiff.js), which is using the `isomorphic-fetch` library.  It would suggest creating an issue at https://github.com/geotiffjs/geotiff.js and see what the package maintainer has to say.  It's possible no one has told them it's an issue yet.

If I'm missing something or I'm wrong, definitely let me know though.  Webpack confuses me sometimes :-)

Reply:
As a workaround, could you run `fromUr`l with `forceXHR` to true.  This should counter-intuitively trigger geotiff.js to use `http.get` by skipping using the failing fetch poyfill and skipping over XMLHttpRequest when it isn't found on node.  Source code for that logic is here: https://github.com/geotiffjs/geotiff.js/blob/f5223455df67a449d403481b7d9d6f1dab8ee054/src/source.js#L374

Reply:
@DanielJDufour Thanks, this has fallen off my radar a bit as well recently. I'll try to make another pass this week and see if I can make progress on getting the test to work as well as any other remaining issues you've mentioned.

Reply:
FYI: I found a fix to this.  I'll be pushing it up within a couple days.  I basically had to set `node-fetch` to a global variable and then geotiff.js picked that up and used makeFetchSource.

I also ran into another error with the COG being UTM, which isn't a problem, because georaster needs utm support anyway :-)